reporting/src/main/scala/caliban/reporting/SchemaReporter.scala

```diff
@@ -0,0 +1,98 @@
+package caliban.reporting
+
+import caliban.client.CalibanClientError.CommunicationError
+import caliban.reporting.ReportingError.{ ClientError, RetryableError }
+import sttp.client3.UriContext
+import sttp.client3.asynchttpclient.zio.SttpClient
+import sttp.client3.asynchttpclient.zio.SttpClient.Service
+import zio.duration.durationInt
+import zio.{ Has, IO, Tag, ZIO, ZLayer }
+import zio.system.System
+
+trait SchemaReporter {
+  def report(ref: SchemaReportingRef, withCoreSchema: Boolean): IO[ReportingError, ReportingResponse]
+}
+
+object SchemaReporter {
+
+  def make(accessToken: String) = (for {
+    client <- ZIO.service[SttpClient.Service]
+  } yield new SchemaReporter {
+    import caliban.reporting.client.{ Mutation, ReportSchemaError, ReportSchemaResponse, SchemaReport }
+
+    private final val REPORTING_URL = "https://schema-reporting.api.apollographql.com"
```

Comment:
Is this always this URL?

Reply:
Yeah it’s a fixed URL, I checked some of the other implementions and they all hard code it